code/web/src/modules/user/ProfileEditor.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import jest from "jest";
+import ProfileEditor from "./ProfileEditor"
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect"
+import { Provider } from "react-redux";
+import { createStore } from "redux";
+
+describe('ProfileEditor', () => {
```

Comment:
I think you're on the right track for setting up these tests in Redux.  I'll share my redux testing code I did on another project to show you how I went about testing this stuff.